pkg/cmd/server/api/v1/conversions.go

```diff
@@ -17,6 +17,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.APILevels) == 0 {
 				obj.APILevels = internal.DefaultOpenShiftAPILevels
 			}
+			if len(obj.JenkinsPipelineConfig.Namespace) == 0 {
+				obj.JenkinsPipelineConfig.Namespace = "openshift"
+			}
+			if len(obj.JenkinsPipelineConfig.TemplateName) == 0 {
+				obj.JenkinsPipelineConfig.TemplateName = "jenkins"
+			}
+			if len(obj.JenkinsPipelineConfig.ServiceName) == 0 {
+				obj.JenkinsPipelineConfig.ServiceName = "jenkins-ci"
```

Comment:
You can't have two auto CI installations in the same project.  If you
aren't using Jenkins to deliver pipelines, you aren't using Jenkins.

On Wed, Mar 30, 2016 at 5:32 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/conversions.go
> <https://github.com/jimmidyson/origin/pull/2#discussion_r57967728>:
>
> > @@ -17,6 +17,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
> >  			if len(obj.APILevels) == 0 {
> >  				obj.APILevels = internal.DefaultOpenShiftAPILevels
> >  			}
> > +			if len(obj.JenkinsPipelineConfig.Namespace) == 0 {
> > +				obj.JenkinsPipelineConfig.Namespace = "openshift"
> > +			}
> > +			if len(obj.JenkinsPipelineConfig.TemplateName) == 0 {
> > +				obj.JenkinsPipelineConfig.TemplateName = "jenkins"
> > +			}
> > +			if len(obj.JenkinsPipelineConfig.ServiceName) == 0 {
> > +				obj.JenkinsPipelineConfig.ServiceName = "jenkins-ci"
>
> what happens when i have both in a single project?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jimmidyson/origin/pull/2/files/3ed08d495dd391ab47317d2946c2695da23b317b#r57967728>
>